packages/components/tabs/src/tabs.tsx

```diff
@@ -95,9 +96,9 @@ export default defineComponent({
       emit('tab-change', value)
     }
 
-    const setCurrentName = async (value: TabPanelName) => {
+    const setCurrentName = async (value: TabPanelName | undefined) => {
```

Comment:
```suggestion
    const setCurrentName = async (value?: TabPanelName) => {
```

Why not?

Reply:
This is taken into account when modifying, but there is no handling of undefined here, so another method is chosen. I will adjust.